app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,84 @@
+class TagsController < ApplicationController
+	before_action :set_tag , only: [:show]
+
+  def index
+
+  end
+
+  def edit
+    @tag = Tag.find(tag_params[:id])
+
+  end
+
+	def show
+
+  end
+
+  def new
+    @tag = Tag.new
+    @file = BruseFile.find(params[:file_id])
+
+  end
+
+  def create
+    
+    @file = BruseFile.find_by(:id => tag_params[:file_id])
```

Comment:
Maybe we should put some kind of check to see if the `current_user` owns the file.
It could be done with adding `if current_user.id == @file.identity.user_id`, and then only if it's true we proceed with adding tags to that file.